detox/android/detox/src/main/java/com/wix/detox/instruments/reflected/InstrumentsReflected.java

```diff
@@ -34,8 +33,6 @@
             methodGetInstanceOfProfiler = profilerClass.getDeclaredMethod("getInstance", Context.class);
             methodStartRecording = profilerClass.getDeclaredMethod("startProfiling", Context.class, configurationClass);
         } catch (Exception e) {
-            Log.i("DetoxInstrumentsManager", "InstrumentsRecording not found", e);
```

Comment:
I hate to be a nag but if we're going to remove the log altogether, better make the catch more explicit (i.e. of a ClassNotFound exception) and _keep_ the log for other exception types